contents/docs/surveys/troubleshooting.mdx

```diff
@@ -54,3 +54,14 @@ You can call `getActiveMatchingSurveys` in your browser console to see which sur
 posthog.getActiveMatchingSurveys(surveys => console.log(surveys))
 ```
 
+## How are surveys priced?
+
+Surveys are priced based on the [number of responses](/pricing?product=surveys). However, because surveys is a tool that utilizes other parts of PostHog, such as events and feature flags, it's important to highlight the underlying costs too.
+
+Whenever a response is sent, it's stored as an event in PostHog, along with survey "shown" and "dismissed" events. Events are free up to the 1st million per month, and then $0.00031 after that. This should be negligible unless you're sending surveys to millions of users a month.
+
+Whenever user properties targeting or linked flags are used in a survey, feature flags are used underneath the hood. Feature flags are free up to the 1st million requests per month, and then $0.0001 after that.
+
+We want to make sure that our customers use surveys without any hidden costs because surveys was built in a way that allows maximum flexibility with the rest of what PostHog offers. For example, because responses are stored as events, you can use PostHog [insights](/docs/features/insights) to analyze your survey responses even further, and to easily export data. Or if you want to create cohorts of users who've responded to a survey in a certain way, you can do that too with cohorts and feature flags.
```

Comment:
```suggestion
### Why does surveys work with events and feature flags?

Building surveys in this way allows maximum flexibility and insight by combining with the rest of what PostHog offers. For example, because responses are stored as events, you can use PostHog [insights](/docs/features/insights) to analyze your survey responses even further, and to easily export data. Or if you want to create cohorts of users who've responded to a survey in a certain way, you can do that with cohorts and feature flags.
```

Reply:
```suggestion
### Why do surveys use events and feature flags?
We built surveys in this way to enable maximum flexibility and ensure it combines well with other PostHog tools. 

Storing responses as events means you can use them within insights or cohorts, for example. Unlike other tools, PostHog doesn't limit you to analyzing responses in a single way. Want to create a feature flag that targets users who responded to a survey in a particular way, or breakdown responses by geography? PostHog does that!
```